packages/playwright-test/src/test.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Is this file for internal testing? Let's be more specific with the name, like `utilitiesForTest`, because `test` is too overloaded 😄

Reply:
True